binding.gyp

```diff
@@ -1,10 +1,16 @@
 {
+  "variables": { 
+    "target_arch": "<!(uname -p)"
```

Comment:
Ok I made that change, thanks again I was unaware of the target_arch variable within gyp.